docs/ROADMAP.md

```diff
@@ -52,53 +52,58 @@ scenarios like limiting the customization and validation to a subset of
 allowed functions.  The function catalog has received additional functions, 
 examples and help. Please visit the [function catalog] for further information.
 
-## In Progress
-
 ### 7. Targeting resources in `kpt fn` commands
 
-Users want to invoke a kpt function (imperatively and declaratively) on a subset of 
-resources in the package by selecting them on the basis of GVKNN, package-path, 
-file path etc. For example, set work-load identity annotation on all Kubernetes 
+Users want to invoke a kpt function (imperatively and declaratively) on a subset of
+resources in the package by selecting them on the basis of GVKNN, package-path,
+file path etc. For example, set work-load identity annotation on all Kubernetes
 Service Account resources in this package [Tracker](https://github.com/GoogleContainerTools/kpt/issues/2015).
-**Estimated release date:** Last week of September 2021.
 
+## In Progress
```

Comment:
I would suggest moving `In Progress` section at the top followed by `Recently Completed` section because as an end-user, I am interesting in seeing what's being worked on or what are the upcoming features.